contracts/shapella-upgrade/ShapellaUpgradeTemplate.sol

```diff
@@ -941,30 +1014,31 @@ contract ShapellaUpgradeTemplate {
 
         IBurner burner = _burner;
         _assertSingleOZRoleHolder(burner, DEFAULT_ADMIN_ROLE, agent);
-        _assertTwoOZRoleHolders(burner, burner.REQUEST_BURN_SHARES_ROLE(), address(_lido), address(_nodeOperatorsRegistry));
+        _assertZeroOZRoleHolders(burner, REQUEST_BURN_MY_STETH_ROLE);
+        _assertTwoOZRoleHolders(burner, REQUEST_BURN_SHARES_ROLE, address(_lido), address(_nodeOperatorsRegistry));
 
         _assertSingleOZRoleHolder(_hashConsensusForAccountingOracle, DEFAULT_ADMIN_ROLE, agent);
         _assertSingleOZRoleHolder(_accountingOracle, DEFAULT_ADMIN_ROLE, agent);
 
         _assertSingleOZRoleHolder(_hashConsensusForValidatorsExitBusOracle, DEFAULT_ADMIN_ROLE, agent);
         IValidatorsExitBusOracle vebo = _validatorsExitBusOracle;
         _assertSingleOZRoleHolder(vebo, DEFAULT_ADMIN_ROLE, agent);
-        _assertZeroOZRoleHolders(vebo, vebo.RESUME_ROLE());
-        _assertSingleOZRoleHolder(vebo, vebo.PAUSE_ROLE(), _gateSeal);
+        _assertZeroOZRoleHolders(vebo, RESUME_ROLE);
+        _assertSingleOZRoleHolder(vebo, PAUSE_ROLE, _gateSeal);
 
         IStakingRouter sr = _stakingRouter;
         _assertSingleOZRoleHolder(sr, DEFAULT_ADMIN_ROLE, agent);
-        _assertZeroOZRoleHolders(sr, sr.STAKING_MODULE_RESUME_ROLE());
-        _assertSingleOZRoleHolder(sr, sr.STAKING_MODULE_PAUSE_ROLE(), address(_depositSecurityModule));
-        _assertSingleOZRoleHolder(sr, sr.REPORT_EXITED_VALIDATORS_ROLE(), address(_accountingOracle));
-        _assertSingleOZRoleHolder(sr, sr.REPORT_REWARDS_MINTED_ROLE(), address(_lido));
+        _assertSingleOZRoleHolder(sr, STAKING_MODULE_PAUSE_ROLE, address(_depositSecurityModule));
+        _assertSingleOZRoleHolder(sr, STAKING_MODULE_RESUME_ROLE, address(_depositSecurityModule));
```

Comment:
Have we settled this change with the tests team?

Reply:
@DiRaiks fyi